### PR TITLE
Update user export

### PR DESCRIPTION
**Pull request by sambuddhabasu**

### CODE REVIEW

**Comment by mitechie** [comment]
Looks good but no tests so will wait to formally review it and qa it.

File: bookie/templates/utils/export.mako [hunk 1]
```diff
@@ -10,7 +10,7 @@
     % for bmark in bmark_list:
         <DT>
             <A HREF="${bmark.hashed.url}" LAST_VISIT="" ADD_DATE="${time.mktime(bmark.stored.timetuple())}"
-               TAGS="${','.join([tag for tag in bmark.tags])}">
+               TAGS="${','.join([tag for tag in bmark.tags])}" IS_PRIVATE="${bmark.is_private}">
```
Comment (mitechie) [inline bookie/templates/utils/export.mako]: If you look at the test files we use for import testing, the key is just PRIVATE and not IS_PRIVATE.

File: bookie/tests/test_utils/test_export.py [hunk 2]
```diff
@@ -84,3 +85,33 @@ def test_export_wo_tags(self):
...
+        self._get_good_request()
+
+        res = self.app.get('/admin/export', status=200)
+
+        self.assertTrue(
+            "google.com" in res.body,
+            msg='Google is in the exported body: ' + res.body)
+
+        self.assertTrue(
+            'IS_PRIVATE="False"' in res.body,
```
Comment (mitechie) [inline bookie/tests/test_utils/test_export.py]: so then this assertion needs to be updated

File: bookie/tests/test_utils/test_export.py [hunk 3]
```diff
@@ -84,3 +85,33 @@ def test_export_wo_tags(self):
...
+        self._get_good_request(is_private=True)
+
+        res = self.app.get('/admin/export', status=200)
+
+        self.assertTrue(
+            "google.com" in res.body,
+            msg='Google is in the exported body: ' + res.body)
+
+        self.assertTrue(
+            'IS_PRIVATE="True"' in res.body,
```
Comment (mitechie) [inline bookie/tests/test_utils/test_export.py]: as does this one.

File: bookie/tests/test_utils/test_export.py (hunk 2 above)
Comment (mitechie) [inline bookie/tests/test_utils/test_export.py]: and in the false case, we should just not apply the private attribute to the item. It should only show if it is.

File: bookie/tests/test_utils/test_export.py (hunk 3 above)
Comment (mitechie) [inline bookie/tests/test_utils/test_export.py]: also note that the attribute is just a 1

```
PRIVATE="1"
```

**Comment by mitechie** [comment]
The attribute it a bit off of what I think it needs to be so that it's the proper format. Please also verify you can import your output into the system as well and get a private bookmark in the system.

**Comment by mitechie** [comment]
:+1: thanks for the update.

**Comment by preddy5** [comment]
:+1:

**Comment by sambuddhabasu** [comment]
:shipit:

**Pull request merged**